packages/tests/package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "test": "npm run test:dev && npm run test:prod",
     "test:dev": "npx wtr",
-    "test:prod": "MODE=prod npm run test:dev",
+    "test:prod": "TEST_PROD_BUILD=true MODE=prod npm run test:dev",
```

Comment:
What does `TEST_PROD_BUILD` mean? I don't see it actually being consumed anywhere in our code (though I do see it's used in other `test:prod` npm scripts).

Reply:
I saw it in the `test` command of a few packages. Was mainly covering my bases. if it's not consumed anywhere, we can get rid of it!

So that's what i'll do :)

Reply:
I removed the TEST_PROD_BUILD from all package.jsons

Since TEST_PROD_BUILD is not used anywhere, those packages are just running DEV tests twice.
Those packages now echo "prod tests identical to dev tests, skipping"

Down to change, but I don't see any reason to run tests twice

Reply:
resolving for now! gonna need another review because some files have changed :)